resources/comments/commentsController.js

```diff
@@ -1,6 +1,8 @@
 const go = require('../utils/crud');
 const db = require('../../data/dbConfig');
 
+const userMatches = require('../utils/userMatches');
```

Comment:
userMatches could use more specific syntac of name. "userComp" userComparison? perhaps. Or at least have a very brief note to explain.